### ISSUE TITLE
Using dart to build version control

### ISSUE THREAD

**Issue by Qharny** [codecrafters-io/build-your-own-x#1280]
### Main programming language
 Dart Langguage

### Tutorial title
Build yout own version control with dart

### Tutorial URL
 → [Steps](https://github.com/Qharny/dartvcs) ←

### Category
* [ ] 3D Renderer
* [ ] Augmented Reality
* [ ] BitTorrent Client
* [ ] Blockchain / Cryptocurrency
* [ ] Bot
* [ ] Command-Line Tool
* [ ] Database
* [ ] Docker
* [ ] Emulator / Virtual Machine
* [ ] Front-end Framework / Library
* [ ] Game
* [ ] Git
* [ ] Network Stack
* [ ] Neural Network
* [ ] Operating System
* [ ] Physics Engine
* [ ] Programming Language
* [ ] Regex Engine
* [ ] Search Engine
* [ ] Shell
* [ ] Template Engine
* [ ] Visual Recognition System
* [ ] Voxel Engine
* [ ] Web Search Engine
* [ ] Web Server
* [ ] Uncategorized

**Comment by FabriLluvia** [comment]
@noam-no
I need your help with this one

**Comment by FabriLluvia** [comment]
@noam-no, Do you think this enters as a tutorial? I don't think so. If you think so, I could add it to my PR, in "Uncategorized".

I think it would be something like `* [**Dart**: _Build yout own version control with Dart_](INSERT URL HERE)
`

**Comment by Qharny** [comment]
@FabriLluvia , That's fine with me

**Comment by noam-no** [comment]
@FabriLluvia yes, good idea.

**Comment by FabriLluvia** [comment]
Yes, but I need the URL, because the one provided here is just a link to the repository, and not to the steps.
